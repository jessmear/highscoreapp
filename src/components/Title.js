import React from 'react'

const Title = props => {
  return (
    <div className='title-box'>
      <h1 className='title'>{props.children}</h1>
    </div>
  )
}

export default Title