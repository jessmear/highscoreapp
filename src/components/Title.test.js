import React from 'react';
import { mount } from 'Enzyme';
import Title from './Title';

const title = 'High Score App';
let wrapped = mount(<Title>{title}</Title>);

describe('renders title', () => {
  it('renders the title component correctly', () => {
    expect(wrapped).toMatchSnapshot();
  });
  it('renders the title children', () => {
    expect(wrapped.find('h1').text()).toEqual(title);
  });
});
