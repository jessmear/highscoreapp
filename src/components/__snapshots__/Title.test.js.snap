// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders title should render the title Component correctly 1`] = `
<div
  className="title-box"
>
  <h1
    className="title"
  >
    High Score App
  </h1>
</div>
`;

exports[`renders title should render the title component correctly 1`] = `
<div
  className="title-box"
>
  <h1
    className="title"
  >
    High Score App
  </h1>
</div>
`;
